scripts/utils-wolfssl.sh

```diff
@@ -41,7 +42,7 @@ install_wolfssl() {
     if [ ! -d ${WOLFSSL_SOURCE_DIR} ]; then
         printf "\tClone wolfSSL ${WOLFSSL_TAG} ... "
         git clone --depth=1 -b ${WOLFSSL_TAG} ${WOLFSSL_GIT} \
-             ${WOLFSSL_SOURCE_DIR} &>> $LOG_FILE
+             ${WOLFSSL_SOURCE_DIR} 2>&1 >> $LOG_FILE
```

Comment:
```
In scripts/utils-wolfssl.sh line 45:
             ${WOLFSSL_SOURCE_DIR} 2>&1 >> $LOG_FILE
             ^-------------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
                                   ^--^ SC2069 (warning): To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).
                                           ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```